packages/apps/templates/bare-template/src/tsconfig.json.t.ts

```diff
@@ -18,7 +18,7 @@ export default defineTemplate<typeof config>(async ({ input, outputDirectory })
     emitDeclarationOnly: false,
     lib: ['DOM', 'ESNext'],
     outDir: 'dist',
-    types: ['node']
+    types: ['node', 'vite-plugin-pwa/client']
```

Comment:
Technically, this is only needed if the user opts in for PWA support. Not sure if there's a better way to selectively add this based on user selection...

We could also drop this change, as this represents defaults, and rely on the individual templates to set it.